spring-integration-redis/src/main/java/org/springframework/integration/redis/store/RedisMessageStore.java

```diff
@@ -37,6 +37,7 @@
  * @author Oleg Zhurakousky
  * @author Gary Russell
  * @author Artem Bilan
+ * @author Youbin Wu
```

Comment:
Looks like no changes in this class, so no need in extra `@author` for nothing.
You have moved the logic in other super classes, and that is really great.